source/dev_section/programming_tutorials/programming_setup/ps.rst

```diff
@@ -15,7 +15,7 @@ Setup
 #. Materials for *Application Tutorial 3* will be provided later.
 
 #. Download the
-   `EnMAP-Box Workshop 2019 Repository <https://bitbucket.org/hu-geomatics/enmap-box-workshop2019/get/develop.zip>`_
```

Comment:
Better leave as was before. This tutorial is pretty outdated (there is no enmap-box-workshop2019 repository)